### PR TITLE
Added translation file for Icelandic

### PR DESCRIPTION
**Pull request by trymbill**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [x] Other, please describe: Translation addition

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

### CODE REVIEW

File: ui/lang/is.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    pluralDay: 'dagar'
+  },
+  table: {
+    noData: 'Engin gögn tiltæk',
+    noResults: 'Engar samsvarandi skrár fundust',
+    loading: 'Augnablik...',
+    selectedRecords: function (rows) {
+      return rows === 1
+        ? '1 færsla valin.'
+        : (rows === 0 ? 'No' : rows) + ' færslur valdar.'
```
Comment (hawkeye64) [inline ui/lang/is.js:35]: Shouldn't the 'No' here be: "Engar"? As in "Engar færslur valdar"?

**Review by hawkeye64 — CHANGES_REQUESTED** [review]
Please look at the review, thanks

File: ui/lang/is.js (hunk 1 above)
Comment (trymbill) [reply ui/lang/is.js:35]: Hot damn, good catch! Missed the inline if there. Correct on the translation, I'll update.

**Comment by hawkeye64** [comment]
Good work, thanks for contributing! :heart:

**Comment by rstoenescu** [comment]
This will go into Quasar v1.15.1.

**Pull request merged**